_tutorials/2016-07-18-python-automation-netmiko-and-napalm.md

```diff
@@ -0,0 +1,226 @@
+---
+published: false
+date: '2016-07-18 23:11 +0300'
+title: 'Python automation: netmiko and napalm'
```

Comment:
Keep your title catchy. The title shows up on Twitter, so to draw attention, mention IOS-XR in the title whenever possible. 
Change this to :  "Using Netmiko and Napalm with IOS-XR: Quick Look"   or something similar